templates/web/northnorthants/tracking_code.html

```diff
@@ -0,0 +1,94 @@
+[% IF c.config.BASE_URL == "https://www.fixmystreet.com" OR c.config.BASE_URL == 'https://staging.fixmystreet.com' %]
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag() {
+    dataLayer.push(arguments);
+  }
+
+  window.onload = () => {
+    const cookieBanner = document.getElementById("wncCookieBanner");
+    if (localStorage.getItem("wncCookieConsentMode") === null) {
+      cookieBanner.classList.remove("wnc-cookie-hidden");
+      cookieBanner.classList.add("wnc-cookie-show");
+    }
+
+    if (localStorage.getItem("wncCookieConsentMode") === null) {
+      gtag("consent", "default", {
+        ad_storage: "denied",
+        analytics_storage: "denied",
+        personalization_storage: "denied",
+        functionality_storage: "denied",
+        security_storage: "denied",
+      });
+    } else {
+      console.log('gtag');
+      gtag(
+        "consent",
+        "default",
+        JSON.parse(localStorage.getItem("wncCookieConsentMode"))
+      );
+    }
+
+    gtag("js", new Date());
+    gtag("config", "G-0Z09LP5XJ7");
+  };
+</script>
+
+<!-- Google tag (gtag.js) -->
+<script
+  async
+  src="https://www.googletagmanager.com/gtag/js?id=G-0Z09LP5XJ7"
+></script>
+
+<script>
+  function updateCookies(consent) {
+    const cookieBanner = document.getElementById("wncCookieBanner");
+    cookieBanner.classList.remove("wnc-cookie-show");
+    cookieBanner.classList.add("wnc-cookie-hidden");
+    setConsent(consent);
+  }
+
+  function setConsent(consent) {
+    const wncCookieConsentMode = {
+      functionality_storage: consent ? "granted" : "denied",
+      security_storage: consent ? "granted" : "denied",
+      ad_storage: "denied",
+      analytics_storage: consent ? "granted" : "denied",
+      personalization: "denied",
+    };
+    gtag("consent", "update", wncCookieConsentMode);
+    localStorage.setItem(
+      "wncCookieConsentMode",
+      JSON.stringify(wncCookieConsentMode)
+    );
+  }
+</script>
+
+<div class="wnc-cookie-banner wnc-cookie-hidden" id="wncCookieBanner">
```

Comment:
Because this template is included in the `<head>` element, this `<div>` can't be here as it ends up outside the `<body>`.

Reply:
Yes, this is in the West code as well, luckily only a few links after it as it auto starts the body when reached.

Reply:
https://github.com/mysociety/fixmystreet/pull/4843 was original PR